pkg/jobs/jobs.go

```diff
@@ -852,25 +867,12 @@ func unmarshalCreatedBy(createdByType, createdByID tree.Datum) (*CreatedByInfo,
 		"job: failed to unmarshal created_by_type as DString (was %T)", createdByType)
 }
 
-// CurrentStatus returns the current job status from the jobs table or error.
-func (j *Job) CurrentStatus(ctx context.Context, txn *kv.Txn) (Status, error) {
```

Comment:
should we keep this method, with a name like "FetchCurrentStatus" or something, since it does something different than the replacement?

Reply:
It had no legitimate users, it was used only in test files. I think not.